Sources/Data/Model/Experiences/BarcodeBlock.swift

```diff
@@ -32,4 +32,50 @@ public struct BarcodeBlock: Block {
         self.keys = keys
         self.tags = tags
     }
+    
+    private enum CodingKeys: String, CodingKey {
```

Comment:
You can omit the `= "..."` part for all cases where the case name is an exact match of the JSON string name. I.e. everything except for `typeName`.